pkg/kubectl/cmd/resize.go

```diff
@@ -65,7 +65,7 @@ func NewCmdResize(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 
 func RunResize(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string) error {
 	count := cmdutil.GetFlagInt(cmd, "replicas")
-	if len(args) != 2 || count < 0 {
+	if count < 0 {
```

Comment:
Need to either handle multiple infos or return an error if there is more than one (down in r.Infos()